packages/dashboards/src/lib/components/timeseries-widget/chart-presets/xy-chart/xy-chart.component.html

```diff
@@ -4,28 +4,61 @@
-                               [descriptionPrimary]="legendSeries.legendDescriptionPrimary"
-                               [descriptionSecondary]="legendSeries.legendDescriptionSecondary"
-                               [isSelected]="!chartAssist.isSeriesHidden(legendSeries.id)"
-                               (isSelectedChange)="chartAssist.toggleSeries(legendSeries.id, $event)"
-                               (mouseenter)="chartAssist.emphasizeSeries(legendSeries.id)"
-                               (mouseleave)="chartAssist.resetVisibleSeries()"
-                               [seriesRenderState]="chartAssist.renderStatesIndex[legendSeries.id]?.state"
-                               (click)="onLegendClick(legendSeries, $event)">
+            <nui-legend-series 
+                    [ngClass]="'legend-container '+ 'state-' + chartAssist.renderStatesIndex[legendSeries.id]?.state"
+                        *ngFor="let legendSeries of chartAssist.legendSeriesSet; trackBy: chartAssist.seriesTrackByFn"
+                        [isSelected]="!chartAssist.isSeriesHidden(legendSeries.id)"
+                        (isSelectedChange)="chartAssist.toggleSeries(legendSeries.id, $event)"
+                        (mouseenter)="chartAssist.emphasizeSeries(legendSeries.id)"
+                        (mouseleave)="chartAssist.resetVisibleSeries()"
+                        [seriesRenderState]="chartAssist.renderStatesIndex[legendSeries.id]?.state"
+                        (click)="onLegendClick(legendSeries, $event)">
                 <nui-rich-legend-tile [value]="chartAssist.getHighlightedValue(legendSeries, 'y', undefined, valueAccessorKey)"
-                                      [backgroundColor]="chartAssist.palette.standardColors.get(legendSeries.id)"
-                                      [color]="chartAssist.palette.textColors.get(legendSeries.id)">
+                            [backgroundColor]="chartAssist.palette.standardColors.get(legendSeries.id)"
+                            [color]="chartAssist.palette.textColors.get(legendSeries.id)">
                     <nui-chart-marker rightEdge
-                                      [marker]="chartAssist.markers.get(legendSeries.id)"
-                                      [color]="chartAssist.palette.standardColors.get(legendSeries.id)">
+                                [marker]="chartAssist.markers.get(legendSeries.id)"
+                                [color]="chartAssist.palette.standardColors.get(legendSeries.id)">
                     </nui-chart-marker>
                 </nui-rich-legend-tile>
+
+                <div description class="description-container d-flex flex-column nui-text-small justify-content-center">
+
+                        <div *ngIf="!legendSeries.link" 
+                            class="description description-primary" 
+                            [ngClass]="{'nui-text-link-small link': isSeriesInteractive(legendSeries.id)}"
```

Comment:
I think the result of isSeriesInteractive may be too vague to know whether these link classes should be applied to the primary description. For example, if there's a secondary link but no primary link, it will return true and add the link styling to the primary description even thought there's no link for the primary.

Reply:
Maybe we should used the `this.seriesInteractive` property instead since that tells us whether there's a urlInteractionHandler?